x/slashing/types.go

```diff
@@ -8,3 +8,10 @@ type SideDowntimeSlashPackage struct {
 	SideChainId   types.ChainID `json:"side_chain_id"`
 	SideTimestamp uint64        `json:"side_timestamp"`
 }
+
+type SideMalicousVoteSlashPackage struct {
+	SideVoteAddr  []byte        `json:"side_vote_addr"`
```

Comment:
The definition is exactly same with `SideDowntimeSlashPackage`, that is why it wont work.